services/subscriber/service.go

```diff
@@ -95,14 +94,10 @@ func (s *Service) Open() error {
 func (s *Service) Close() error {
```

Comment:
This should probably have a comment stating that calling `Close()` multiple times will panic.